cookbook/case_studies/house_price_prediction/sandbox.config

```diff
@@ -0,0 +1,7 @@
+[sdk]
+workflow_packages=house_price_prediction
+python_venv=flytekit_venv
+
+[auth]
+assumable_iam_role=arn:aws:iam::173840052742:role/flytefunctionaltestsbatchworker-production-iad
```

Comment:
Can we get rid of this line?  I don't think it's necessary.  Can you try running without this line?  If it doesn't work, can you try replacing this line with `kubernetes_service_account=default`

Reply:
The code is working locally without assumable_iam_role.